rules/windows/registry/registry_set/registry_set_add_load_service_in_safe_mode.yml

```
@@ -30,6 +30,9 @@ detection:
         Image|endswith: '\MBAMInstallerService.exe'
         TargetObject|endswith: '\MBAMService\(Default)'
         Details: 'Service'
+    filter_optional_hexnode:
+        Image|endswith: 'C:\Hexnode\Hexnode Agent\Current\HexnodeAgent.exe'
+        Details: 'Service'
```

```suggestion
        Details: 'Service'
```
I think we should also add TargetObject here just to be more specific

---

thanks. i have applied the filter accordingly.